doc/guides/releases.md

```diff
@@ -747,6 +747,14 @@ For example:
 
 The changes must be made as part of a new semver-minor release.
 
+### Update release labels
+
+The `lts-watch-vN.x` issue label must be created, with the same color as other
+existing labels for that release line, such as `vN.x`.
+
+If the release is transitioning from Active LTS to Maintenance, the
+`backport-requested-vN.x` label must be deleted.
```

Comment:
Not when it is transitioning out of Maintenance?

Reply:
I think this is to avoid getting PRs like https://github.com/nodejs/node/pull/34950.